test/testdrive/protobuf-basic.td

```diff
@@ -0,0 +1,68 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Test that basic types in a Protobuf message are decoded correctly.
+
+$ file-append path=basic.proto
+syntax = "proto3";
+
+message Basic {
+    enum Enum {
+        ENUM0 = 0;
+        ENUM1 = 1;
+    }
+
+    message Message {
+        int32 m1 = 1;
+        int32 m2 = 2;
+    }
+
+    bool bool = 1;
+    int32 int32 = 2;
+    int64 int64 = 3;
+    float float = 4;
+    double double = 5;
+    bytes bytes = 6;
+    string string = 7;
+    Enum enum = 8;
+    Message message = 9;
+}
+
+$ protobuf-compile-descriptors inputs=basic.proto output=basic.pb
+
+$ kafka-create-topic topic=basic partitions=1
```

Comment:
Looks like one thing that was dropped when splitting out / rewriting files was multi-partition topics.

If this was intentional, can you drop a short note explaining why it's okay?  It's not immediately obvious to me

Reply:
Ah yeah, just didn't seem like a necessary thing to test. There's nothing special about protobuf that would interact weirdly with multi-partition topics, and we've got plenty of multi-partition topics elsewhere.

Reply:
makes sense!